src/views/Accessibility/index.tsx

```diff
@@ -112,6 +112,15 @@ const RequestDetails = (
   />
 );
 
+const RequestDetailsNotes = (
```

Comment:
This route is unmounting on every link change. This is causing the app to refetch the data every single time the user changes between documents and notes. Ideally, the route shouldn't unmount. You probably need to update the `path` prop so that the route always matches.